src/common/common.go

```diff
@@ -215,24 +219,24 @@ func HostMatchByNameOrIPAddress(node v1.Node, namesMap, IPAddressMap map[string]
 // Returns True when uninitialized taint must be removed.
 // This is required for some external platforms (e.g. VSphere, Nutanix) to proceed
 // with the installation using fake credentials.
-func RemoveUninitializedTaint(platform *models.Platform, invoker string, hasValidCredentials bool, openshiftVersion string) bool {
+func RemoveUninitializedTaint(platformType models.PlatformType) bool {
```

Comment:
Sorry, i think i was pretty not clear :( 
i actually meant to run hasValidCredentials function under https://github.com/openshift/assisted-installer/pull/896/files#diff-a5bd908493da50da6a2905424525321bb4b7b81b90da798952ae8aac3ec53546L221
and not to create 2 functions, WDYT?

Reply:
Sorry for my misunderstanding. Let me convert it back to a single function and have hasValidvSphereCredentials be called within RemoveUniniitializedTaint.